### PR TITLE
TACKLE-575: Uses exposed ingress limit to validate uploaded binaries

### PR DESCRIPTION
**Pull request by ibolton336**
image

Closes https://issues.redhat.com/browse/TACKLE-575?filter=12395291

### CODE REVIEW

File: pkg/client/src/app/Constants.ts [hunk 1]
```diff
@@ -19,9 +19,11 @@ export interface IEnvVars {
   KEYCLOAK_REALM: string;
   KEYCLOAK_CLIENT_ID: string;
   KEYCLOAK_SERVER_URL: string;
+  UI_INGRESS_PROXY_BODY_SIZE: string;
 }
 export const ENV: IEnvVars = window["_env"] || {};
 export const isAuthRequired = ENV.AUTH_REQUIRED !== "false";
+export const uploadLimit = ENV.UI_INGRESS_PROXY_BODY_SIZE || "500m";
```
Comment (amastbau) [inline pkg/client/src/app/Constants.ts:26]: I think MB or MiB is preferred.wdty?
@ibolton336

**Review by amastbau — CHANGES_REQUESTED** [review]

File: pkg/client/src/app/Constants.ts (hunk 1 above)
Comment (ibolton336) [reply pkg/client/src/app/Constants.ts:26]: @amastbau https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#custom-max-body-size Going off of the format used from the ingress annotation here. It seems that they use 'm' to indicate megabytes & not sure there is a way to change it.

Comment (ibolton336) [reply pkg/client/src/app/Constants.ts:26]: @jortel @fbladilo

**Comment by mturley** [comment]
@ibolton336 while you're in here can you set the `variant` of the modal to be medium or large? It looks funky taking up the full screen width for a short message.

**Review by mturley — APPROVED** [review]

**Pull request merged**